extension/popup.js

```diff
@@ -155,8 +155,15 @@ function showAddToListSuccessMessage(tab, url) {
     return geti18nMessages(url.origin, [ MESSAGE_KEYS.success ])
     .then(messages => getCanonicalPageTitle(tab)
```

Comment:
Not related to the patch, but the brackets are a bit confusing here. There isn't any difference between

    .then(messages => getCanonicalPageTitle(tab))
    .then(...)

and

    .then(messages => getCanonicalPageTitle(tab)
        .then(...)
    )

as far as I can see, so the first version could be used instead; otherwise, it would be nice if the indenting were more regular. .

Reply:
Addressed code formatting in 826724bf.